packages/semver/src/builders/version/utils/workspace.spec.ts

```diff
@@ -50,53 +51,48 @@ describe('getPackageFiles', () => {
       })
     );
 
-    expect(await getPackageFiles('/root').toPromise()).toEqual([
+    expect(
+      await getPackageFiles(['/root/packages/a, /root/packages/b']).toPromise()
+    ).toEqual([
       '/root/packages/a/package.json',
       '/root/packages/b/package.json',
     ]);
-
-    expect(fs.readFile).toBeCalledTimes(1);
-    expect(fs.readFile).toBeCalledWith(
-      '/root/workspace.json',
-      'utf-8',
-      expect.any(Function)
-    );
   });
 
-  it('should fallback to angular.json if workspace.json is not found', async () => {
-    fakeReadFileSync.mockImplementationOnce(() => {
-      throw new Error('ENOENT, no such file or directory');
-    });
-    fakeReadFileSync.mockReturnValue(
-      JSON.stringify({
-        version: 1,
-        projects: {
-          a: {
-            root: 'packages/a',
-          },
-          b: {
-            root: 'packages/b',
-          },
-        },
-      })
-    );
+  // it('should fallback to angular.json if workspace.json is not found', async () => {
```

Comment:
Wouldn't it be better to just exclude the test `xit` to disable it temporarily if there is something in progress? If it doesn't compile yet, then maybe we could comment out just the code inside. Otherwise, we could forget it ;)